rules/solid/tests/Rector/Foreach_/ChangeNestedForeachIfsToEarlyContinueRector/Fixture/multi_exprs_with_bool_oprnds.php.inc

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Rector\SOLID\Tests\Rector\Foreach_\ChangeNestedForeachIfsToEarlyContinueRector\Fixture;
+
+use Rector\Core\Testing\Contract\RunnableInterface;
+
+class MultiExprsWithBoolOprnds implements RunnableInterface
+{
+    public function run()
+    {
+        $loaded = true;
+        $usePackages = true;
+        $notBad = true;
+        $partPackageList = [ new \StdClass(), null, 'string', 0, [] ];
+        $payload = [];
+
+        foreach( $partPackageList as $partPackage )
+        {
+            if(! is_null( $partPackage ) && ! is_object($partPackage) )
+            {
+                if ($loaded || ($usePackages && $notBad))
+                {
+                    $payload[] = $partPackage;
+                }
+            }
+        }
+
+        return json_encode($payload);
```

Comment:
The snippet is quite large to cover with test only the problematic spot. 

2 variables and ~5 lines of code in method would be ideal.

<br>

What lines can be remove to still show the bug?

Reply:
Is it better now?

Reply:
Better. Is that the least code needed?

E.g.?

```diff
-$partPackageList = [ new \StdClass(), null, 'string', 0, [] ];
+$partPackageList = [];
```

Reply:
I've stripped it down as much as I can while reproducing the bug.

Reply:
Great, thanks!